mappings/net/minecraft/client/render/item/BuiltinModelItemRenderer.mapping

```diff
@@ -13,5 +13,6 @@ CLASS net/minecraft/class_756 net/minecraft/client/render/item/BuiltinModelItemR
 	METHOD method_3166 render (Lnet/minecraft/class_1799;Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;II)V
 		ARG 1 stack
 		ARG 2 matrix
+		ARG 3 vertexConsumers
```

Comment:
vertices or vertex consumers?

Reply:
I don't even know at this point. There have been so many changes and retcons to these things...